api/src/helpers/formatError.ts

```diff
@@ -1,28 +1,12 @@
-import { unwrapResolverError } from '@apollo/server/errors';
 import { GraphQLFormattedError } from 'graphql';
-import { isInstance } from 'apollo-errors';
-import { AuthenticationError, NotFoundError } from '../errors/errors.js';
+import { NotFoundError } from '../errors/errors.js';
 
 function myFormatError(formattedError: GraphQLFormattedError, error: Error): GraphQLFormattedError {
-  // if (unwrapResolverError(error) instanceof CustomDBError) {
-  //   return { message: 'Internal server error' };
-  // }
   if (error.message.startsWith('DataLoader must be constructed')) return new NotFoundError('Id Not Found')
   if (error.message.startsWith('Context creation failed: ')) {
     error.message = error.message.replace('Context creation failed: ', '')
   }
 
-  if (error.message.startsWith('AuthenticationError: ')) return new AuthenticationError({ data: { unexpected: error.message.replace('AuthenticationError: ', '') } })

```

Comment:
was this removed intentionally?

Reply:
yes, I don't believe there should be any more Authentication Errors

Reply:
Ah ok, yeah makes sense